app/helpers/forms_helper.rb

```diff
@@ -69,4 +69,14 @@ def render_public_contact(public_contact_form, object)
            object: object,
            provider_form: object.is_a?(Provider)
   end
+
+  def action_prompt(object, action)
+    entities = object == "catalogue" ? "providers and services" : "services"
```

Comment:
Maybe we should also add offers here because they are also deleted

Reply:
Sure

Reply:
@Marcelinna that can be discussed and agreed upon, but its a separate talk I believe. For now, offers are just deleted permanently. What can be done in this PR is to inform users that offers will also be deleted

Reply:
Now it's done.

Reply:
add bundles